core/src/test/java/io/questdb/test/cutlass/pgwire/BasePGTest.java

```diff
@@ -64,6 +64,10 @@
 public abstract class BasePGTest extends AbstractCairoTest {
 
     protected CopyRequestJob copyRequestJob = null;
+    protected int forceRecvFragmentationChunkSize = 1024 * 1024;
```

Comment:
Might be a good idea to reset the new fields to their defaults in `setUp()` or `tearDown()` to minimize test side effects.

Reply:
I'd rather avoid it. Some tests call super.setUp(), super.tearDown() and they should not reset the buffer-related params in them.

Reply:
Sorry, I don't think I follow. All tests within the same test suite should start in the same setup, independently of the test execution order. If we don't reset these parameters, that won't be the case. Also, a well-behaved test class should call super methods in its `setUp()`/`tearDown()` overrides. Am I missing anything?

Reply:
Let's consider this one to be a nit.